pkgs/development/haskell-modules/configuration-common.nix

```
@@ -1854,8 +1854,13 @@ self: super: {
   # https://github.com/adnelson/semver-range/issues/15
   semver-range = dontCheck super.semver-range;
 
-  # 2022-06-19: Disable checks because of https://github.com/reflex-frp/reflex/issues/475
-  reflex = doJailbreak (dontCheck super.reflex);
+  reflex = lib.pipe super.reflex [
+    doJailbreak
+    # Until hackage release has https://github.com/reflex-frp/reflex/pull/517
+    (overrideCabal (drv: {
+      postPatch = "sed -i 's/Data.Witherable/Witherable/' src/Data/*.hs src/Reflex/*.hs src/Reflex/Spider/*.hs test/*.hs";
```

1. Make sure not discard any other `postPatch` changes elsewhere (hypothetic ones in this case).
2. Using `--replace-fail` is better here since it won't fail silently when the problem is fixed upstream.

This could be achieved like this (untested):

```suggestion
      postPatch = drv.postPatch or "" + ''
        substituteInPace src/Data/*.hs src/Reflex/*.hs src/Reflex/Spider/*.hs test/*.hs --replace-fail Data.Witherable Witherable
      ''
```

---

Could not use `*` with `--replace-fail`, so I listed individual files.

---

Aha! That's why my own code always uses `for` loops with it.